source/docs/articles/sites/code/hot-fixes.md

```diff
@@ -2,8 +2,10 @@
 title: Hot Fixes
 description: Learn how to deploy a hotfix on Pantheon.
 category:
-  - developing
+  - developing
 ---
+For Experts only. You should not need to attempt this if you use [Multidev](/docs/articles/sites/multidev) and keep commits from reaching Dev that you do not intend on deploying.

```

Comment:
What is the CSS class for making this a bootstrap alert? This should smell of a bad idea from first sight